addons/account/models/partner.py

```diff
@@ -285,6 +285,11 @@ def _journal_item_count(self):
             partner.journal_item_count = self.env['account.move.line'].search_count([('partner_id', '=', partner.id)])
             partner.contracts_count = self.env['account.analytic.account'].search_count([('partner_id', '=', partner.id)])
 
+    @api.multi
```

Comment:
@antonylesuisse Here also use api.multi. Why we use read_group or sql query here. When bank_ids is one2many field that count bank_ids is easily get using len().
is it required to use read_group here ?